### PR TITLE
Small patches on Docs (Tutorials and FAQ)

### PR DESCRIPTION
**Pull request by bingyao**
## PR Summary

4 commits, one to each file.

### CODE REVIEW

File: tutorials/text/text_intro.py [hunk 1]
```diff
@@ -353,7 +353,6 @@
 #  Finally, we can specify functions for the formatter using
 # `matplotlib.ticker.FuncFormatter`.
 
-
```
Comment (ImportanceOfBeingErnest) [inline tutorials/text/text_intro.py]: This line needs to stay to make PEP8 happy.

Comment (bingyao) [reply tutorials/text/text_intro.py]: I see. Thanks for reminding! And it seems that another blank line is also needed after this function definition.

**Comment by timhoffm** [comment]
Thanks!

**Comment by QuLogic** [comment]
@meeseeksdev backport to v3.1.x

**Pull request merged**